app/js/controllers/foyer/ressources/ressources.js

```diff
@@ -72,16 +89,39 @@ angular.module('ddsApp').controller('FoyerRessourcesCtrl', function($scope, $sta
         if (individu.caMicroEntreprise) {
             result.push({
                 type: _.find(ressourceTypes, { id: 'caMicroEntreprise' }),
-                tnsStructureType: individu.tnsStructureType,
-                tnsActiviteType: individu.tnsActiviteType,
+                tnsActiviteType: individu.microEntrepriseActiviteType,
                 montantAnnuel: individu.caMicroEntreprise
             });
         }
 
+        if (individu.caAutoEntrepreneur) {
+
+            var montantsMensuels = _.map($scope.months, function(month) {
+                var ressource = _.find(ressources, { periode: month.id, type: 'caAutoEntrepreneur' });
+                return ressource ? Math.round(ressource.montant) : 0;
```

Comment:
Default the value in the fetch rather than in the `return` statement. That will make your intentions clearer.

I.e. `ressource = _.find(…) || { montant: 0 }`, and always return `Math.round(ressource.montant)`.